From c816a572a95cff1f4790b04636135e262b1ba405 Mon Sep 17 00:00:00 2001
From: Chau Ly <chaul@amperecomputing.com>
Date: Tue, 17 Sep 2024 13:34:36 +0700
Subject: [PATCH] managers: pid fan: Ignore AccumulateSetPoint

Add AccumulateSetPoint to the skip list of FanZones in
/redfish/v1/Managers/bmc/Oem/OpenBmc/Fan/FanZones.

Signed-off-by: Chau Ly <chaul@amperecomputing.com>
---
 redfish-core/lib/managers.hpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/redfish-core/lib/managers.hpp b/redfish-core/lib/managers.hpp
index 6d4dc6e5..494f2261 100644
--- a/redfish-core/lib/managers.hpp
+++ b/redfish-core/lib/managers.hpp
@@ -541,7 +541,8 @@ inline void
                     {
                         if (propertyPair.first == "Type" ||
                             propertyPair.first == "Class" ||
-                            propertyPair.first == "Name")
+                            propertyPair.first == "Name" ||
+                            propertyPair.first == "AccumulateSetPoint")
                         {
                             continue;
                         }
-- 
2.25.1

